aspnetcore/fundamentals/startup.md

```diff
@@ -74,6 +74,10 @@ A common use of [dependency injection](xref:fundamentals/dependency-injection) i
 
 [!code-csharp[](startup/sample_snapshot/Startup2.cs?highlight=7-8)]
 
+Most services are not available until the `Configure` method is called.
+
+::: moniker-end
+
 An alternative to injecting `IHostingEnvironment` is to use a conventions-based approach.
```

Comment:
If you're going this way ... which is good btw since "a conventions-based approach" is a little unclear until the reader gets to the next paragraphs ... I recommending deleting Line 81 ...

> An alternative to injecting `IHostingEnvironment` is to use a conventions-based approach.

... but if you do keep that line, then it has to go inside the moniker-block on Line 79.

Reply:
Might want to add a section heading tho ... or something. This is abrupt going into the convention-based bits next here.